oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -946,6 +985,9 @@ export default class Map extends React.PureComponent {
         this.infoRequest.abort();
       }
       this.infoOverlay.setPosition(undefined);
+      if (e.selected[0].get("type") == "area") {
+        this.selectedFeatures.clear();
+      }
```

Comment:
Clear the selectedFeatures object so that users can click on the same area to reopen the plot.